lib/elixir/lib/code/normalizer.ex

```diff
@@ -227,11 +232,29 @@ defmodule Code.Normalizer do
     end
   end
 
+  # Tuples
+  defp normalize_literal({:{}, _, args} = quoted, meta, state) do
+    last_arg = List.last(args)
+
+    with [{{:__block__, key_meta, _}, _} | _] <- last_arg, :keyword <- key_meta[:format] do
+      {:__block__, meta, [quoted]}
+    else
+      _ ->
+        normalize_call(quoted, state)
+    end
+  end
+
   # 2-tuples
   defp normalize_literal({left, right}, meta, state) do
     meta = patch_meta_line(meta, state.parent_meta)
     state = %{state | parent_meta: meta}
-    {:__block__, meta, [{do_normalize(left, state), do_normalize(right, state)}]}
+
+    with [{{:__block__, key_meta, _}, _} | _] <- right, :keyword <- key_meta[:format] do
+      {:__block__, meta, [{do_normalize(left, state), right}]}
```

Comment:
We still need to normalize the right, no? For example, we need to normalize the values of the keyword.